traffic_portal/app/src/common/modules/navigation/navigation.tpl.html

```diff
@@ -48,6 +48,7 @@
                         <li class="side-menu-category-item" ng-if="hasCapability('params-read')" ng-class="{'current-page': isState('trafficPortal.private.parameters')}"><a href="/#!/parameters">Parameters</a></li>
                         <li class="side-menu-category-item" ng-if="hasCapability('types-read')" ng-class="{'current-page': isState('trafficPortal.private.types')}"><a href="/#!/types">Types</a></li>
                         <li class="side-menu-category-item" ng-if="hasCapability('statuses-read')" ng-class="{'current-page': isState('trafficPortal.private.statuses')}"><a href="/#!/statuses">Statuses</a></li>
+                        <li class="side-menu-category-item" ng-if="hasCapability('service-categories-read')" ng-class="{'current-page': isState('trafficPortal.private.serviceCategories')}"><a href="/#!/servicecategories">Service Categories</a></li>
```

Comment:
I'm kinda thinking this belongs under the Services section as the 3rd item. Plus the `Configure` section is getting lengthy.

Reply:
you want to make this change?